sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentenceSentiment.java

```diff
@@ -14,8 +14,8 @@
     private final String text;
     private final TextSentiment sentiment;
     private final SentimentConfidenceScores confidenceScores;
-    private final IterableStream<MinedOpinion> minedOpinions;
-    private final int offset;
+    private IterableStream<MinedOpinion> opinions;
```

Comment:
I am unsure about the use of `final` in Java, but pointing out here in case it was a mistake and needs to be added again.
Same for `length`

Reply:
`final` means once the parameter's value is assigned. It can't change. For creating a object, it is required when the object is initialized.